libgnucash/app-utils/test/gtest-gnc-quotes.cpp

```diff
@@ -150,7 +155,8 @@ TEST_F(GncQuotesTest, quote_sources)
 
 TEST_F(GncQuotesTest, quotable_commodities)
 {
-    auto commodities{gnc_quotes_get_quotable_commodities(gnc_commodity_table_get_table(m_book))};
+    auto commtable{gnc_commodity_table_get_table(m_book)};
+    auto commodities{gnc_quotes_get_quotable_commodities(commtable)};
```

Comment:
Why create the extra variable?

Reply:
No reason other than the line getting a bit long.
But I can remove the change if you want, as it is not related to the memleak fix.

Reply:
It was shorter than lines 109 and 119 are now that you removed the newlines.